docs/migration/supported-versions.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
I think this content would make more sense as part of the Getting Started page instead of a new separate page, especially for visibility. I have an idea for this that I'll add for you to review.